easy_access/db/compat.py

```
@@ -0,0 +1,553 @@
+                                ):
+                                    row["file_exists"] = 1 if row["file_exists"] else 0
+                    elif isinstance(batch, dict):
+                        if "in_collection" in batch and isinstance(
+                            batch["in_collection"], bool
+                        ):
+                            batch["in_collection"] = 1 if batch["in_collection"] else 0
+                        if "file_exists" in batch and isinstance(
+                            batch["file_exists"], bool
+                        ):
+                            batch["file_exists"] = 1 if batch["file_exists"] else 0
+                except Exception:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)